rts/lra/test/pom.xml

```diff
@@ -196,16 +224,113 @@
               </execution>
             </executions>
           </plugin>
+
+          <!-- Makes sure that the two Wildfly instances use different standalone XML configurations -->
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>${version.antrun.plugin}</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <phase>generate-test-resources</phase>
+                <configuration>
+                  <target>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.coordinator.xml.filename}"></copy>
+                    <copy file="${env.JBOSS_HOME}/standalone/configuration/standalone.xml" tofile="${env.JBOSS_HOME}/standalone/configuration/${lra.participant.xml.filename}"></copy>
+                  </target>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+
+          <!--
+            This profile needs to kick in when Wildfly is used to run this TCK.
+            The reason is that lra-coordinator is not in the LRA specification
+            and, therefore, a scenario with two deployments in a single instance
+            of Wildfly (or distributed in two Wildfly instances) is not compatible
+            with the LRA TCK.
```

Comment:
I need to change this as I am not more using a profile to start Wildfly through `wildfly-maven-plugin`